### ISSUE TITLE
Custom Card Not Registering

### ISSUE THREAD

**Issue by mdframe** [moralmunky/Home-Assistant-Mail-And-Packages-Custom-Card#29]
After the latest release I am unable to resolve the error "Custom element doesn't exist: mail-and-packages-card."

I have removed all my cache, I have tried browsers and systems that have never touch my Home Assistant instance.   I have removed and re-added all the components several times however I can no longer get the custom card to be recognized.  

Where should I look for information to help debug this custom card error?  I am using the latest beta version, V0.7.2.3, and it is managed by HACS in my implementation.

**Comment by moralmunky** [comment]
That's the million dollar question. I haven't been able to make heads or tails of those situations and they magically resolve themselves sometimes. 

When you remove it from HACS, Check that all js references are removed from the Configuration > Lovelace dashboard > resources area. Then go through the cache and restart install dance.

**Comment by D34DC3N73R** [comment]
I've just run into this myself on v0.7.2.7. I've uninstalled, removed it from resources, restarted, confirmed it's not in /config/www/community. Then reinstalled, add the path back to resources, restart, clear browser cache as well as incognito mode + other browsers, a separate computer, and phone, and still have the same issue. I'm running in yaml mode.

configuration.yaml
```
lovelace:
  mode: yaml
  resources: !include resources.yaml
```
resources.yaml
```
- url: /hacsfiles/Home-Assistant-Mail-And-Packages-Custom-Card/Home-Assistant-Mail-And-Packages-Custom-Card.js
  type: module
```
I have lots of other resources all working fine with this setup. The only one giving me issues is this mail and packages card.

**Comment by D34DC3N73R** [comment]
ok, I think I've realized what the issue is. 
v0.7.2.7 uses `-type: 'custom:mailandpackages-card'` while v0.6 uses `-type: 'custom:mail-and-packages-card'`

The problem I'm having is HACS only shows the 4 most recent betas, all of which are 0.7+, and I can't go back to the 0.4 release because I need the gif_sensor integration. Would it be possible to release 0.6 as an official release?

**Comment by moralmunky** [comment]
I’ve unchecked 0.6 as a pre-release. See if it shows up for you?

Noted in the first 0.7 series it is recommended to remove the card. It show shows in the card picker and pre-fills the visual card editor with minimal yaml.

Add an issue to add manual set up instructions to the doc for yaml only people?

**Comment by D34DC3N73R** [comment]
0.6 shows up now with beta versions unchecked. I manually replaced the JS files in the meantime and I was able to overwrite them with a reinstall of 0.6 through HACS without issue. 

After switching from yaml to storage mode I did see the custom card in the card library with the 0.7+ versions, but it was always blank, even after entering sensor data (also noticed the on/off toggle logic appeared to be reversed). I figured that may just be part of why it's beta, but I can open a separate issue for that if needed. When I get the 0.7+ versions working I can submit a suggested yaml mode solution.
